matrix/matrix_class.py

```diff
@@ -101,10 +101,25 @@ class Matrix:
      [198. 243. 288. 304.]
      [306. 378. 450. 472.]
      [414. 513. 612. 640.]]
-
+    >>> rows1 = [[1], [2], [3]]
+    >>> rows2 = [[3], [2], [1]]
+    >>> vector1 = Matrix(rows1)
+    >>> vector2 = Matrix(rows2)
+    >>> print(vector1.cross_product(vector2))
+    [[-4.]
+     [8.]
+     [-4.]]
+    >>> rows1 = [[2], [1]]
+    >>> rows2 = [[5], [4]]
+    >>> vector1 = Matrix(rows1)
+    >>> vector2 = Matrix(rows2)
+    >>> print(vector1.cross_product(vector2))
+    [[0.]
+     [0.]
+     [3.]]
     """
 
-    def __init__(self, rows):
+    def __init__(self, rows: list):

```

Comment:
Please use generic types from the `typing` module: `typing.List[type]`, `typing.Tuple[type, ...]` and please provide the return type hint as well. In case the function does not contain a return statement, it implicitly returns `None`